gidgethub/routing.py

```diff
@@ -62,12 +61,12 @@ def decorator(func: AsyncCallback) -> AsyncCallback:
 
         return decorator
 
-    async def dispatch(self, event: sansio.Event, *args: Any, **kwargs: Any) -> None:
-        """Dispatch an event to all registered function(s)."""
-
-        found_callbacks = []
+    def fetch(self, event: sansio.Event) -> Set[AsyncCallback]:
```

Comment:
A frozenset should be returned so as to prevent people breaking encapsulation and mutating the set directly.